README.md

```diff
@@ -12,15 +12,41 @@ Accounts are disabled or deleted when they did not log in within the given numbe
 * ⛔ Exclude accounts based on group memberships (default: admin group)
 * 🔑 Exclude accounts that never logged in (default: enabled)
 
+As an administrator, click on your user icon, then navigate to 
+**Administration Settings** -> **Basic settings** -> **Account retention**.
+
 ![Screenshot of the admin settings](docs/screenshot.png)
 
-## 🔐 Accounts that never logged in
+## Configuration options
+
+There are a few configuration options to be aware of, which can be set and 
+retrieved via the occ command.
+
+For example, to disable users after 10 days of inactivity:
+ 
+```shell
+occ config:app:set user_retention user_days_disable --value='10'
+```
+
+| Configuration key          | Default value      | Description                                                                                                 |
+|----------------------------|--------------------|-------------------------------------------------------------------------------------------------------------|
+| `user_days_disable`        | `0`                | If greater than `0`, disables users who have been inactive for the specified number of days.                |
+| `user_days`                | `0`                | If greater than `0`, deletes users who have been inactive for the specified number of days.                 |
+| `guest_days_disable`       | `0`                | If greater than `0`, disables guest users who have been inactive for the specified number of days.          |
+| `guest_days`               | `0`                | If greater than `0`, deletes guest users who have been inactive for the specified number of days.           |
+| `reminder_days`            | `''` (empty value) | Comma-separated list of days before which reminder emails are sent about upcoming deactivation or deletion. |
+| `keep_users_without_login` | `yes`              | When set to `yes`, preserves users who have never logged in.                                                |
+| `excluded_groups`          | `["admin"]`        | List of groups whose members are excluded from deactivation and deletion policies.                          |
```

Comment:
I planned to migrate to the IAppConfig Lexicons as soon as they are finally OCP and  then we could document those with their type properly.

Reply:
Now the types in the readme match the ones in the code :)